src/test/kotlin/uk/gov/dluhc/printapi/messaging/SendApplicationToPrintMessageListenerIntegrationTest.kt

```diff
@@ -41,7 +110,7 @@ internal class SendApplicationToPrintMessageListenerIntegrationTest : Integratio
         await.during(5, SECONDS).until {
             assertThat(
                 TestLogAppender.hasLog(
-                    "Sending application [${payload.sourceReference}] to print", Level.INFO
+                    "Print message with source reference [${payload.sourceReference}] received", Level.INFO
```

Comment:
Any thoughts on changing this assertion to assert that a record has not been written to dynamo, rather than asserting that log messages have not been written?

(I'm not a fan on asserting business logic based on the presence or not of log messages - I know why we did it initially because the service and repo etc where not there, and thats fine. But the implementation now allows us to do a better assertion ?)

Reply:
Thanks, for the comment, @NathanRussellValtech .
I have changed the assertion to assert no data has persisted.

Reply:
Yeah, thats much better. Thank you 🙏